CHANGELOG.next.toml

```diff
@@ -102,3 +102,15 @@ message = "Required members with @contextParam are now treated as client-side re
 references = ["smithy-rs#2964"]
 meta = { "breaking" = false, "tada" = false, "bug" = false, target = "client" }
 author = "rcoh"
+
+[[aws-sdk-rust]]
+message = "Port redacting sensitive HTTP response bodies to the orchestrator"
+references = ["smithy-rs#2926", "smithy-rs#2972"]
+meta = { "breaking" = false, "tada" = false, "bug" = true }
+author = "ysaito1001"
+
+[[smithy-rs]]
+message = "Port redacting sensitive HTTP response bodies to the orchestrator"
```

Comment:
Consider calling this, "Fix regression with redacting sensitive HTTP response bodies"

Reply:
+1. Addressed in 590e28f.